extensions/team-time/src/index.tsx

```diff
@@ -0,0 +1,252 @@
+                    if (t.name === time.name) {
+                      t.favorite = false;
+                      t.favoritePosition = 0;
+                    }
+                    return t;
+                  });
+                  // Reorder the favorites
+                  newTimes
+                    .filter((t: TimeEntry) => t.favorite)
+                    .sort((a: TimeEntry, b: TimeEntry) => a.favoritePosition - b.favoritePosition)
+                    .forEach((t: TimeEntry, index: number) => {
+                      t.favoritePosition = index + 1;
+                    });
+
+                  await LocalStorage.setItem("times", JSON.stringify(newTimes));
+                  revalidate();
+                }}
+              />
+            )}
+            {time.favorite && time.favoritePosition > 1 && (
+              <Action
+                title="Move up in Favorites"
+                icon={Icon.ArrowUp}
+                shortcut={{ modifiers: ["opt", "cmd"], key: "arrowUp" }}
+                onAction={async () => {
+                  // Find the time that is above this one
+                  const timeAbove = times.find((t: TimeEntry) => t.favoritePosition === time.favoritePosition - 1);
+                  const newTimes = times.map((t: TimeEntry) => {
+                    if (t.name === time.name) {
+                      t.favoritePosition -= 1;
+                    } else if (t.name === timeAbove?.name) {
+                      t.favoritePosition += 1;
+                    }
+                    return t;
+                  });
+                  await LocalStorage.setItem("times", JSON.stringify(newTimes));
+                  revalidate();
+                }}
+              />
+            )}
+            {time.favorite && time.favoritePosition < times.filter((t: TimeEntry) => t.favorite).length && (
+              <Action
+                title="Move down in Favorites"
+                icon={Icon.ArrowDown}
+                shortcut={{ modifiers: ["opt", "cmd"], key: "arrowDown" }}
+                onAction={async () => {
+                  // Find the time that is below this one
+                  const timeBelow = times.find((t: TimeEntry) => t.favoritePosition === time.favoritePosition + 1);
+                  const newTimes = times.map((t: TimeEntry) => {
+                    if (t.name === time.name) {
+                      t.favoritePosition += 1;
+                    } else if (t.name === timeBelow?.name) {
+                      t.favoritePosition -= 1;
+                    }
+                    return t;
+                  });
+                  await LocalStorage.setItem("times", JSON.stringify(newTimes));
+                  revalidate();
+                }}
+              />
+            )}
+          </ActionPanel.Section>
+          <ActionPanel.Section title="Team Time">
+            <Action.Push title="Add New Time" icon={Icon.AddPerson} target={<AddTime onAdd={revalidate} />} />
```

Comment:
Wording is a bit confusing here, also maybe add ⌘+n for new?

Reply:
I have renamed it to "Add Teammember" and added the Shortcut! Thanks for the suggestion